### PR TITLE
Add patch bank account

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

File: spec/paths/bank-accounts@{id}.yaml [hunk 1]
```diff
@@ -24,6 +24,58 @@ get:
...
+            address:
+              description: The Address.
+              allOf:
+                - $ref: "#/components/schemas/ContactObject"
+            customFields:
+              $ref: "#/components/schemas/ResourceCustomFields"
+    description: Bank Account
+  responses:
+    200:
+      description: Payment card was updated
```
Comment (XaosSintez) [inline spec/paths/bank-accounts@{id}.yaml]: I guess it's `Bank Account`

File: spec/paths/bank-accounts@{id}.yaml [hunk 2]
```diff
@@ -24,6 +24,58 @@ get:
...
+        Rate-Limit-Limit:
+          $ref: "#/components/headers/Rate-Limit-Limit"
+        Rate-Limit-Remaining:
+          $ref: "#/components/headers/Rate-Limit-Remaining"
+        Rate-Limit-Reset:
+          $ref: "#/components/headers/Rate-Limit-Reset"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/PaymentCard"
```
Comment (XaosSintez) [inline spec/paths/bank-accounts@{id}.yaml]: `BankAccount`

**Comment by XaosSintez** [comment]
Is this for some upcoming changes?

**Comment by ashkarpetin** [comment]
@XaosSintez 
Yes, we need the ability to update at least billing address

File: spec/paths/bank-accounts@{id}.yaml [hunk 3]
```diff
@@ -24,6 +24,58 @@ get:
...
+              description: Bank's name.
+              type: string
+            accountType:
+              description: Bank's Account type.
+              type: string
+              enum:
+                - checking
+                - savings
+                - other
+            bic:
```
Comment (ivizma) [inline spec/paths/bank-accounts@{id}.yaml]: I would remove BIC from update.
The purpose of BIC is kind of checksum for IBAN but if we don't have access to IBAN we can't check it against BIC anymore.

**Pull request merged**